src/main/java/frc/robot/Robot.java

```diff
@@ -22,7 +21,6 @@
 
   @Override
   public void robotInit() {
-    Logger.DEBUG = true;
```

Comment:
Why is `DEBUG` off?

Reply:
I deprecated `Logger.DEBUG` and is unused. I deprecated it as the logs can just be ignored during Comp so no reason to have a flag to disable it